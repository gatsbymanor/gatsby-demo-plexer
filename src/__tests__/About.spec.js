import React from 'react';
import renderer from 'react-test-renderer';
import About from '../pages/About';

describe('<About />', () => {
  const component = shallow(
    <About />
  );

  it('should render correctly', () => {
    expect(component).toMatchSnapshot();
  });

});
