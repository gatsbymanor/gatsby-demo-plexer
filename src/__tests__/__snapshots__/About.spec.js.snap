// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<About /> should render correctly 1`] = `
<div>
  <h1>
    About
  </h1>
  <p>
    Here is an about page.
  </p>
  <GatsbyLink
    to="/"
  >
    Home
  </GatsbyLink>
</div>
`;
